web/client/src/library/components/editor/EditorInspector.tsx

```diff
@@ -110,17 +120,37 @@ function InspectorModel({
         <Tab.Panel
           unmount={false}
           className={clsx(
-            'text-xs w-full h-full ring-white ring-opacity-60 ring-offset-2 ring-offset-blue-400 focus:outline-none focus:ring-2 py-2',
+            'text-xs w-full h-full ring-white ring-opacity-60 ring-offset-2 ring-offset-blue-400 focus:outline-none focus:ring-2 p-2',
           )}
         >
-          <Documentation
-            model={model}
-            withCode={false}
-            withModel={false}
-            withQuery={tab.file.isSQLMeshModelSQL}
-            withDescription={false}
+          <ModelColumns
+            className="max-h-[15rem]"
+            nodeId={model.name}
+            columns={model.columns}
+            disabled={model?.type === 'python'}
```

Comment:
Do nodeId and columns need optional chaining on model too?

Reply:
`model` should be present so `model?.type` here does not chaining